README.md

```diff
@@ -49,7 +49,7 @@ To perform the tests locally you'll need the `API_KEY` and `HOST_URL` environmen
 
 | Name                   | Description                                                                 | Required | Default Value |
 |------------------------|-----------------------------------------------------------------------------|----------|---------------|
-| `apiKey`            | The Pinecone api key.                          | Yes      |               |
+| `apiKey`            | The Pinecone API key.                          | Yes      |               |
 | `host`            | The Pinecone index host.                          | Yes      |               |
 | `namespace`            | The Pinecone namespace to target. Defaults to the default Pinecone namespace                           | No      |               |
```

Comment:
@alarbada just spotted this

> Defaults to the default Pinecone namespace

As a user, how could I know which one is my Pinecone namespace?

We need to also indicate its default value comes from `opencdc.collection`. Similar to the kinesis's PR.

Reply:
Hi @raulb. The default pinecone namespace is not configurable from the user, it is a separate entity.

> When you don’t specify a namespace name for an operation, Pinecone uses the default namespace name of "" (the empty string).
from [here](https://docs.pinecone.io/guides/indexes/use-namespaces)

I agree that I should document better this. In other connectors I also pushed for erroring on no collection found, but in this case I can just say that the default namespace will be used instead.

Otherwise, I need to put a special string like "(default)" to allow users to write to the default namespace.